hadoop-hdds/common/src/main/java/org/apache/hadoop/hdds/scm/net/NetworkTopology.java

```diff
@@ -242,4 +242,10 @@ Node getNode(int leafIndex, String scope, List<String> excludedScopes,
    */
   List<? extends Node> sortByDistanceCost(Node reader,
       List<? extends Node> nodes, int activeLen);
+
+  default int getRackCount() {
```

Comment:
Should we have the concept of Racks in Network Topology? Should this particular function go in PlacementPolicy class instead.

Reply:
@sodonnel I remember us discussing this before.